inlong-sdk/sort-sdk/src/main/java/org/apache/inlong/sdk/sort/api/SortClientConfig.java

```diff
@@ -311,6 +328,13 @@ public enum ConsumeStrategy {
         lastest_absolutely
     }
 
+    public enum TopicManagerType {
+        // single topic manager
+        single_topic,
```

Comment:
The enum fields should use uppercase letters.

Reply:
fixed, thx